File: example/src/main/java/autodagger/example/StandardActivityComponent1.java
package autodagger.example;

import autodagger.AutoComponent;

/**
 * @author Lukasz Piliszczuk - lukasz.pili@gmail.com
 */
@Deprecated
@AutoComponent(
        dependencies = MyApp.class,
        superinterfaces = {HasDependenciesOne.class, HasDependenciesTwo.class},
        modules = StandardModule.class
)
public @interface StandardActivityComponent1 {
}

File: example/src/main/kotlin/autodagger/example/second/StandardActivityComponent1.kt
package autodagger.example.second

import autodagger.AutoComponent
import autodagger.example.KotlinExampleApplication
import autodagger.example.first.HasDependenciesOne
import autodagger.example.first.HasDependenciesTwo
import dagger.Module

@AutoComponent(
    dependencies = [KotlinExampleApplication::class],
    superinterfaces = [HasDependenciesOne::class, HasDependenciesTwo::class],
    modules = [StandardModule::class]
)
annotation class StandardActivityComponent1

@Module
class StandardModule